pkg/controllers/restore/controller.go

```diff
@@ -369,6 +370,11 @@ func (h *handler) generateDependencyGraph(ownerToDependentsList map[string][]res
 		}
 
 		metadata := resourceData.Object[metadataMapKey].(map[string]interface{})
+		// ignore the secrets section in the service account to avoid missing token secret error
+		if resourceData.Object["kind"].(string) == "ServiceAccount" {
+			delete(currRestoreObj.Data.Object, secretsMapKey)
+			logrus.Debugf("The secret section is ignored from the service account %s", name)
```

Comment:
```suggestion
			logrus.Debugf("ServiceAccount secret ignored  %s", name)
```

Reply:
This reads to be that secret name follows rather than SA's name. `ServiceAccount secret ignored, SA name: %s` maybe?

Reply:
done.